test/e2e/fixtures/aut.py

```diff
@@ -87,3 +89,10 @@ def user_account(request) -> UserAccount:
 def main_screen(user_account: UserAccount, main_window: MainWindow) -> MainWindow:
     main_window.authorize_user(user_account)
     return main_window
+
+@pytest.fixture(scope="session")
+def close_apps_before_start():
+    running_pids = get_pid_by_process_name('nim_status_client')
```

Comment:
We can add 2 constants here actually and a condition, so if Win then it will be `Status.exe`, in other cases it is `nim_status_client`

Reply:
Done. Added it here